autoload/multiple_cursors.vim

```diff
@@ -317,6 +325,17 @@ function! s:Cursor.remove_visual_selection() dict
   let self.visual_hi_id = 0
 endfunction
 
+" Restore unnamed register from paste buffer
+function! s:Cursor.save_paste_buffer() dict
```

Comment:
Technically we don't have "paste register", we have "paste buffer" (our concept) and "unnamed register" (vim's concept). How about we name it `restore_unnamed_register`?